### PR TITLE
blogs of introducing chinese website

### PR DESCRIPTION
**Pull request by lisa0314**
@bspencer , please help to review. The patch consists of blogs of Chinese and English version.

### CODE REVIEW

File: public/blog/_data.json [hunk 1]
```diff
@@ -183,5 +183,15 @@
...
+  },
+  "chinese-web-release": {
+    "title": "Crosswalk Website has been released!",
+    "date": "2016-05-20T12:00",
+    "author": "Lisha"
+  },
+  "chinese-web-release_zh": {
+    "title": "Crosswalk中文网站发布啦！",
+    "date": "2016-05-20T11:00",
+    "author": "Lisha"
```
Comment (bspencer) [inline public/blog/_data.json]: Why not your Chinese name?

File: public/blog/_data.json [hunk 2]
```diff
@@ -183,5 +183,15 @@
     "title": "Intel XDK: Develop Once, Deploy Everywhere",
     "date": "2014-03-03T11:00",
     "author": "Fredrick Odhiambo"
+  },
```
Comment (bspencer) [inline public/blog/_data.json]: New entries go at the top

File: public/blog/_data.json [hunk 3]
```diff
@@ -183,5 +183,15 @@
...
+  "chinese-web-release": {
+    "title": "Crosswalk Website has been released!",
+    "date": "2016-05-20T12:00",
+    "author": "Lisha"
```
Comment (bspencer) [inline public/blog/_data.json]: First and Last name has been our standard process.  There may be dozens of Lisha's :)

File: public/blog/chinese-web-release.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+The Chinese version of Crosswalk website has been released! The structure of Chinese website is similar with the English version, mainly consisting of four parts: [文档](https://crosswalk-project.org/documentation/getting_started_zh.html), [博客](https://crosswalk-project.org/blog/), [贡献](https://crosswalk-project.org/contribute/index_zh.html) and [维基](https://github.com/crosswalk-project/crosswalk-website/wiki). To accomplish the swift of language version, you are
+required to click the language button at the top right of the page and then select the language you wish.
```
Comment (bspencer) [inline public/blog/chinese-web-release.md]: Chinese characters in the English version is hard to understand for us unenlightened readers.

File: public/blog/chinese-web-release_zh.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+同时，在您浏览网站的过程中也可以随时切换到不同的语言版本。虽然参与中文网站的同学们都在加紧准备相关中文教程，但是目前中文教程内容仍然不够完整。我们会尽快将内容完善，并整合到中文网站中。
+
+
+请大家对Crosswalk中文网站今后的发展多多期待。如果在使用Crosswalk中文网站中发现了什么问题，请通过以下途径反馈给我们：
+1.  过Crosswalk jira提issue。
+2.  过Crosswalk-help mailing list发送邮件。
+
+谢谢大家对Crosswalk的支持，我们会继续努力！
+
```
Comment (bspencer) [inline public/blog/chinese-web-release_zh.md]: Until we have the blog working for Chinese and English, how about we create a single blog posting with both English and Chinese like the "Game Mode" blog?   Then when we have the split working we can create duplicates.

File: public/blog/chinese-web-release_zh.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+
+
+
+
```
Comment (bspencer) [inline public/blog/chinese-web-release_zh.md]: Remove extra lines, generally.

Comment (bspencer) [inline public/blog/chinese-web-release_zh.md]: The 2nd picture above wasn't linked anywhere?  Does it need to be part of the commit?

File: public/blog/_data.json (hunk 3 above)
Comment (lisa0314) [inline public/blog/_data.json]: Yes, I understand. I will replace this with Lisha Guo.

File: public/blog/_data.json (hunk 1 above)
Comment (lisa0314) [inline public/blog/_data.json]: My Chinese name is also called Lisha :)

File: public/blog/chinese-web-release_zh.md (hunk 5 above)
Comment (lisa0314) [inline public/blog/chinese-web-release_zh.md]: ok, I will merge them into one blog ,just like "Game Mode".

**Comment by lisa0314** [comment]
@bspencer  thanks for your review. All suggestions done.

**Pull request merged**